generators/server/templates/src/main/java/package/config/_UaaConfiguration.java

```diff
@@ -87,9 +91,8 @@ public void configure(ClientDetailsServiceConfigurer clients) throws Exception {
          */
         clients.inMemory()
             .withClient("web_app")
-            .scopes("web-app")
+            .scopes("openid")
```

Comment:
this change will currently make the client app fail to communicate with the server. 
In angular client there is a "hack" line, where the Basic Auth header is set incode, rather then using Base64. This is done because I initally wanted to avoid base64 encoding on client side. But as the basic auth header is done once for token request, you should apply it there with base64 encode. I would also suggest to set "openid" as angular.constant, so this is easily configurable, if the developer wants to use a different clientId